test/specs/languages/test-elixir.el

```diff
@@ -0,0 +1,77 @@
+;;; test-elixir.el --- Flycheck Specs: Elixir -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2016 Sebastian Wiesner and Flycheck contributors
+
+;; Author: Kári Tristan Helgason <kari@kthelgason.org>
+
+;; This file is not part of GNU Emacs.
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; Specs for Elixir support.
+
+;;; Code:
+
+(require 'flycheck-buttercup)
+
+(describe "Language Elixir"
+          (describe "The Dogma error parser"
```

Comment:
The indentation is wrong.

Please install and load buttercup, mark the whole buffer and run "indent-region".